3.10/release-notes-api-changes310.md

```diff
@@ -25,4 +28,6 @@ integrations for ArangoDB 3.10.
 
 ### Endpoints removed
 
+* `/_admin/metrics`, metrics API v1 was deprecated in 3.8. Now we removed it, and `/_admin/metrics` just redirect you to `/_admin/metrics/v2`.
```

Comment:
```suggestion
Since ArangoDB 3.8, there were two APIs for retrieving the metrics in two different formats: `/_admin/metrics` and `/_admin/metrics/v2`. The metrics API v1 (`/_admin/metrics`) was deprecated in 3.8 and usage of `/_admin/metrics/v2` was encouraged.  
In ArangoDB 3.10, `/_admin/metrics` and `/_admin/metrics/v2` now behave identically and return the same output in a fully Prometheus-compatible format. The old metrics format is not available anymore.
```